sdk/communication/azure-communication-rooms/azure/communication/rooms/aio/_rooms_client_async.py

```diff
@@ -297,7 +239,7 @@ async def remove_participants(
         self,
         *,
         room_id: str,
-        participant_identifiers: List[CommunicationIdentifier],
+        participants: List[Union[RoomParticipant, CommunicationIdentifier]],
```

Comment:
Why did we switch to union of RoomParticipant and CommunicationIdentifer? Is it mainly to add mroe flexibility to users and is it okay if that deviates from other languages?

Reply:
Yes - it's mostly because I preferred to have consistency between `add_or_update_participants` and `remove_participants`.

Reply:
But I forgot to update the docstrings - I'll do that now.